jicofo/rootfs/defaults/jicofo.conf

```diff
@@ -22,6 +23,11 @@ jicofo {
       {{ else }}
       login-url = "{{ .Env.XMPP_DOMAIN }}"
       {{ end }}
+      {{ if $ENABLE_AUTO_LOGIN }}
```

Comment:
You can write this as `enable-auto-login = {{ $ENABLE_AUTO_LOGIN }}`

Reply:
Got it, Thanks @saghul